app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityPresenter.kt

```diff
@@ -20,8 +24,9 @@ class AdministratorControlsActivityPresenter @Inject constructor(
   private lateinit var navigationDrawerFragment: NavigationDrawerFragment
   private var isMultipane = false
   private lateinit var lastLoadedFragment: String
-  private lateinit var binding: AdministratorControlsActivityBinding
+  private val ADMINISTRATOR_CONTROLS_BACKSTACK: String? = null // "ADMINISTRATOR_CONTROLS_BACKSTACK"
```

Comment:
tried to add a specific tag for the fragments which we are adding to back stack like ProfileListFragment and ProfileEditFragment so that we can push or pop fragments from the back stack instead of replacing fragments.

Reply:
this is working